components/footer/index.css

```diff
@@ -0,0 +1,85 @@
+/*************************************************************************
+* ADOBE CONFIDENTIAL
+* ___________________
+*
+* Copyright 2019 Adobe
+* All Rights Reserved.
+*
+* NOTICE: All information contained herein is, and remains
+* the property of Adobe and its suppliers, if any. The intellectual
+* and technical concepts contained herein are proprietary to Adobe
+* and its suppliers and are protected by all applicable intellectual
+* property laws, including trade secret and copyright laws.
+* Dissemination of this information or reproduction of this material
+* is strictly forbidden unless prior written permission is obtained
+* from Adobe.
+**************************************************************************/
+
+
+:root {
+  --spectrum-footer-height : var(--spectrum-global-dimension-size-500);
+  --spectrum-footer-text-size : var(--spectrum-global-dimension-font-size-75);
+
+  --spectrum-footer-item-gap : 32px;
```

Comment:
Should this be `--spectrum-global-dimension-size-400` so it gets bigger for the large scale?

If it shouldn't change size, you can make it `--spectrum-global-dimension-static-size-400` instead of writing `32px`.

Reply:
no idea, xd file didn't specify and i don't know where the value was picked up from, i'll make it global 400 for now